tests/e2e/e2e_test.go

```diff
@@ -499,9 +501,11 @@ var _ = ginkgo.Describe("[Start/Remove/Restart/Add/Stop]", func() {
 
 	ginkgo.It("can fetch status", func() {
 		ctx, cancel := context.WithTimeout(context.Background(), 15*time.Second)
-		_, err := cli.Status(ctx)
+		resp, err := cli.Status(ctx)
 		cancel()
 		gomega.Ω(err).Should(gomega.BeNil())
+		// get paused node URI for pause node test later
+		pausedNodeURI = resp.ClusterInfo.NodeInfos["node1"].Uri
```

Comment:
pls also check that node1 is key of to NodeInfos

Reply:
are you using this? should use to check that node is fisrt online, then offline, then online